docs/08_cli.md

```diff
@@ -201,3 +201,24 @@ To reset multiple workflows, you can use batch reset command:
 ```
 ./cadence workflow reset-batch --input_file <file_of_workflows_to_reset> --reset_type <reset_type> --reason "some_reason"
 ```
+#### Recovery from bad deployment -- auto-reset workflow
+If a bad deployment lets a workflow run into a wrong state, you might want to reset the workflow to the point that the bad deployment started to run. But usually it is not easy to find out all the workflows impacted, and every reset point for each workflow. In this case, auto-reset will automatically reset all the workflows given a bad deployment identfier. 
+
+First of all, let's get familar with some concepts. Each deployment will have an identifier, we call it "**Binary Checksum**" as it is usually generated by the md5sum of a binary file. For a workflow, each binary checksum will be associated with an **auto-reset point**, which contains a **runID**, an **eventID**, and the **created_time** that binary/deployment made the first decision for the workflow.
```

Comment:
Remove "First of all, let's get familiar with some concepts." The sentence has no value.